examples/python/standalone/weaviate-semantic-search.py

```diff
@@ -0,0 +1,134 @@
+# PREREQUISITE:
+#   Start up Weaviate locally as described on the [Docker Compose](https://weaviate.io/developers/weaviate/installation/docker-compose)
```

Comment:
I think I'd like to commit a docker compose instead of asking them to create it. If weaviate's url changes or the docker compose jumps major versions this will break.